src/chat/chatter.py

```diff
@@ -387,6 +387,7 @@ def viewReplay(self):
     @QtCore.pyqtSlot()
     def viewVaultReplay(self):
         """ see the player replays in the vault """
+        self.lobby.client.replays.setCurrentIndex(2)  # focus on Online Fault
```

Comment:
well, if I would have moved the line, then the question would have been why I move the line.
I can't win here.